library/src/main/java/com/datatorrent/lib/window/Quantification.java

```diff
@@ -0,0 +1,31 @@
+package com.datatorrent.lib.window;
+
+/**
+ * Created by siyuan on 6/3/16.
+ */
+public class Quantification
+{
+  public static interface Unit
+  {
+  }
+
+  public static enum CountableUnit implements Unit
+  {
+    TUPLE,
+    KTUPLE,
+    MTUPLE
+  }
+
+  public static enum TimeUnit implements Unit
```

Comment:
Why are we defining our own TimeUnits instead of using something relatively standard like JodaTime or Java TimeUnit class?